### PR TITLE
feat: Add a section about Daphne in the Django getting started

### LINKED ISSUE #2474: When using the Daphne CLI, the `--proxy-headers` flag should be provided
It is described here: https://github.com/django/daphne/issues/193
It could be a good idea to add some documentation about daphne as it is quite popular lately

### PR DESCRIPTION
**Pull request by SCedricThomas**
fix #2474

### CODE REVIEW

File: src/_posts/languages/python/django/2000-01-01-start.md [hunk 1]
```diff
@@ -100,6 +100,18 @@ Then update `Procfile`:
 web: bash bin/start.sh
 ```
 
+### Using Daphne
+
+It is also possible to use Daphne as an ASGI server.
```
Comment (EtienneM) [inline src/_posts/languages/python/django/2000-01-01-start.md:105]: suggestion: maybe add a link to the Daphne homepage?

File: src/_posts/languages/python/django/2000-01-01-start.md [hunk 2]
```diff
@@ -100,6 +100,18 @@ Then update `Procfile`:
...
+
+Here an example start.sh script to use Daphne:
```
Comment (EtienneM) [inline src/_posts/languages/python/django/2000-01-01-start.md:107]: issue: 
- the sentence is not really well written. It's really hard to understand. I don't think it's a very good English.
- it's kind of hard to understand you refer to the `start.sh` script from the previous section
- I am not sure it should be a different paragraph than the previous one.
question: How is it possible to install the `daphne` CLI?

```suggestion
It is also possible to use Daphne as an ASGI server. You can update the `bin/start.sh` script described in the previous section to use Daphne to start the application:
```

File: src/_posts/languages/python/django/2000-01-01-start.md [hunk 3]
```diff
@@ -100,6 +100,18 @@ Then update `Procfile`:
...
+
+```sh
+#!/bin/bash
+
+daphne -b 0.0.0.0 -p $PORT myapp.asgi:application --proxy-headers
```
Comment (EtienneM) [inline src/_posts/languages/python/django/2000-01-01-start.md:112]: question: are there long options to replace `-b` and `-p`?

**Review by EtienneM — CHANGES_REQUESTED** [review]
Did you tested on the Django sample that it works?

**Comment by SCedricThomas** [comment]
Here is an example of the django sample using Daphne as AGSI server: https://sample.osc-fr1.scalingo.io/

**Review by EtienneM — COMMENTED** [review]
Sorry, just one thing I forgot about in the previous review. Can you update the front matter to update the  last modification date of the page? Then it LGTM

**Comment by SCedricThomas** [comment]
Oh right :face_holding_back_tears:

**Review by EtienneM — APPROVED** [review]

**Pull request merged**